### ISSUE TITLE
Deprecation: chrome.loadTimes() is deprecated

### ISSUE THREAD

**Issue by pekim** [SOASTA/boomerang#182]
The following messages appear in the developer console when using the navtiming plugin in Chrome.
```
[Deprecation] chrome.loadTimes() is deprecated, instead use standardized API: nextHopProtocol in Navigation Timing 2. https://www.chromestatus.com/features/5637885046816768.
[Deprecation] chrome.loadTimes() is deprecated, instead use standardized API: Navigation Timing 2. https://www.chromestatus.com/features/5637885046816768.
[Deprecation] chrome.loadTimes() is deprecated, instead use standardized API: Paint Timing. https://www.chromestatus.com/features/5637885046816768.
```
Chrome version : 64

There is a Chrome specific API used at https://github.com/SOASTA/boomerang/blob/master/plugins/navtiming.js#L190, which is deprecated. I suspect that it will be removed entirely at some point. Although I can't see from either of the links below in which milestone that will happen.

https://www.chromestatus.com/features/5637885046816768
https://bugs.chromium.org/p/chromium/issues/detail?id=621512

**Comment by bluesmoon** [comment]
This is not an issue. We use feature detection to check if the feature exists and if it does, we use it. This is necessary as the feature might be deprecated in the current version of chrome but it is certainly not deprecated in older versions of chrome which we still support.  In future versions where the feature is dropped, we will automatically stop using it.  No code changes are required in boomerang. We will probably only drop support for this feature a few years after the browser has dropped it in order to ensure that no user anywhere is using a user agent that still supports it.

**Comment by bluesmoon** [comment]
keeping this open to avoid dups.

**Comment by querymetrics** [comment]
If we can get the data from NT2, then we shouldn't look at `chrome.loadTimes` which will avoid the console messages in newer Chrome

**Comment by bluesmoon** [comment]
we still need `chrome.loadTimes` for older versions of chrome

**Comment by bluesmoon** [comment]
This is fixed by #188
